util/config/file.go

```diff
@@ -22,9 +22,10 @@ import (
 	"runtime"
 )
 
-// DirectoryPath returns the absolute path to the application's directory.
+// DirectoryPath returns the absolute path to the application's directory
+// in the user standard config location.
 func DirectoryPath() string {
-	switch goos := runtime.GOOS; goos {
+	switch runtime.GOOS {
```

Comment:
hmm, this is almost the same as https://github.com/digitalbitbox/bitbox-wallet-app/blob/master/frontends/qt/server/server.go#L138 :thinking: 

Can we delete this and use backend.arguments.MainDirectoryPath() instead when creating the manager in https://github.com/digitalbitbox/bitbox-wallet-app/pull/16/files#diff-1190f94ab2ad6f04f64211d316e67a04R476 ?

Reply:
I tried by there are too many `logging.Get()` calls.
So, I proposed the reverse: remove `getAppFolder` and use `config.AppDir` instead, in https://github.com/digitalbitbox/bitbox-wallet-app/pull/20